README.md

```diff
@@ -34,6 +34,9 @@ These are all environment variables needed by the application:
 | NODOPERPM_URI                                  | NodoPerPM connection URI                                                           | string |         |
 | NODOPERPM_READ_TIMEOUT                         | NodoPerPM read timeout                                                             | number |         |
 | NODOPERPM_CONNECTION_TIMEOUT                   | NodoPerPM connection timeout                                                       | number |         |
```

Comment:
Maybe there is missing the `TRANSACTION_CLOSURE_SENT_EVENT_QUEUE_NAME` system property?

Reply:
Good catch! Fixed in 43dc59d